SunshineGameFinder/Program.cs

```diff
@@ -116,15 +126,43 @@
         }
         Console.WriteLine(""); //blank line to separate platforms
     }
-    Logger.Log("Finding Games Completed!");
-    if (gamesAdded > 0)
+
+    var logicalDrives = DriveInfo.GetDrives();
+    var wildcatDriveLetter = new Regex(Regex.Escape(wildcatDrive));
+
+    foreach (var drive in logicalDrives)
+    {
+        var libraryFoldersPath = drive.Name + steamLibraryFolders;
+        var file = new FileInfo(libraryFoldersPath);
+        if (!file.Exists)
+        {
+            Logger.Log($"{file.FullName} does not exist, skipping...", LogLevel.Warning);
+            continue;
+        }
+        var libraries = VdfConvert.Deserialize(File.ReadAllText(libraryFoldersPath));
+        foreach(var library in libraries.Value)
+        {
+            if (library is not VProperty libProp)
```

Comment:
Can you comment what exactly this would be skipping? I'm not super familiar with VDF